tests/util/test_udf_functions.py

```diff
@@ -126,6 +123,22 @@ def test_get_output_multiple_matches(self):
                 == "INT64"
             )
 
+        def test_get_output_without_as(self):
+            # note that the odd string formatting below is intentional and spacing affects test results
+            assert (
+                get_output(
+                    """
+                    CREATE OR REPLACE FUNCTION norm.get_windows_info(os_version STRING)
+RETURNS STRUCT<name STRING, version_name STRING, version_number DECIMAL, build_number INT64>
+LANGUAGE js AS r\"\"\"
+  const test = "test";
+  return test;
+\"\"\";
+                    """
+                )
+                == "STRUCT<name STRING, version_name STRING, version_number DECIMAL, build_number INT64> LANGUAGE js"
```

Comment:
I'm not really happy with this test. It's not the result we would want, so I would change the `get_output` function to give us the correct result here